b9lab-content/3-main-concepts/08-modules.md

```diff
@@ -16,19 +16,19 @@ Developers compose modules together using the Cosmos SDK to build custom applica
 
 Modules include **core** functionality that provides the basic functionality every blockchain node needs:
 
-* a boilerplate implementation of the Application Blockchain Interface (ABCI) that communicates with the underlying Tendermint consensus engine
-* a general-purpose data store that persists the module state called `multistore`
-* a server and interfaces to facilitate interactions with the node.
+* A boilerplate implementation of the Application Blockchain Interface (ABCI) that communicates with the underlying Tendermint consensus engine.
+* A general-purpose data store that persists the module state called `multistore`.
+* A server and interfaces to facilitate interactions with the node.
 
 Modules implement the majority of application logic while **core** attends to wiring and infrastructure concerns and enables modules to be composed into higher-order modules.
 
 A module defines a subset of the overall state using one or more key/value stores, known as `KVStore`, and a subset of message types that are needed by the application and do not exist yet. Modules also define interactions with other modules that do already exist.
 
-For developers, most of the work involved in building an SDK application revolves around building custom modules required by their application that do not exist yet, and integrating them with modules that do already exist into one coherent application.
+For developers, most of the work involved in building a Cosmos SDK application revolves around building custom modules required by their application that do not exist yet, and integrating them with modules that do already exist into one coherent application. Existing modules can come either from the Cosmos SDK itself, or **come from third-party developers** and be downloaded from an online module repository.
 
 ## Module Components
 
-It is a best practice to define a module in the `x/moduleName` folder, not to be confused with the SDK’s `x/` folder that already exists. For example, the module called Checkers would go in `x/Checkers`.
+It is a best practice to define a module in the `x/moduleName` folder, not to be confused with the Cosmos SDK’s `x/` folder that already exists. For example, the module called Checkers would go in `x/Checkers`.
```

Comment:
Already exists where, exactly? Let's try to be more clear on the distinction.

Reply:
Updated.